src/lib/parts.ts

```diff
@@ -249,16 +249,16 @@ export class NodePart implements Part {
 
   private _commitTemplateResult(value: TemplateResult): void {
     const template = this.templateFactory(value);
-    let instance: TemplateInstance;
     if (this.value && this.value.template === template) {
-      instance = this.value;
+      this.value.update(value.values);
     } else {
       // Make sure we propagate the template processor from the TemplateResult
       // so that we use it's syntax extension, etc. The template factory comes
       // from the render function so that it can control caching.
-      instance =
+      const instance =
```

Comment:
This should come after elements are upgraded, right before `this._commitNode(fragment);`. That should make the tests pass.

Reply:
A render test in Firefox is still breaking. I'm looking into it

Reply:
We don't run Sauce tests on PRs, so FF is the only configuration that tests the polyfills right now. It's probably also failing on Edge on Sauce.